integration-tests/src/test/resources/wdt-models/coherence-managed-wdt-config.yaml

```diff
@@ -40,20 +48,19 @@ topology:
         'cluster-1-template':
             ListenPort: 8001
             Cluster: 'cluster-1'
-            JTAMigratableTarget:
-                Cluster: 'cluster-1'
-            SSL:
-                ListenPort: 8100
+            CoherenceClusterSystemResource: CoherenceCluster
+            CoherenceMemberConfig:
+              UnicastListenAddress:  'coherence-managed-domain-cluster-1-managed-server${id}' 
         'cluster-2-template':
             ListenPort: 8001
             Cluster: 'cluster-2'
-            JTAMigratableTarget:
-                Cluster: 'cluster-2'
-            SSL:
-                ListenPort: 8100
+            CoherenceClusterSystemResource: CoherenceCluster
+            CoherenceMemberConfig:
+              UnicastListenAddress:  'coherence-managed-domain-cluster-2-managed-server${id}' 
```

Comment:
same comment as above

Reply:
this is a temporary workaround, till we resolve OWLS-91266. This section will be removed once we resolve OWLS-91266.